pom.xml

```diff
@@ -139,6 +139,54 @@
                 <version>3.14.0</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
```

Comment:
Waitwat, we need to add a dependency on `ant` to use java9/java10 ‽
We're living in a sick sad crazy world…

Reply:
So the idea on this PR is to make the libraries that were builtin to the JDK as maven dependencies. If we just took Java 9 as our sole target, we could have enabled these modules (javax.xml.bind, javax.xml.soap, etc). However, lets consider Java 11. A lot / all of the Java EE modules were actually removed so "just enable the modules" solution for Java 9 is no longer valid. Hence the best solution being to include these maven coordinates for the Java EE libraries. 

Now it's not just quite as simple as including these libraries because we have to choose a proper maven coordinate. The challenge is that previous to this attempt many of the already included dependencies included some of these Java EE libraries. So now we have a situation were we need to declare a dependency on the Java EE libraries and some of our existing libraries also have a dependency and chances are the version of two two will not match.

When this happens, our dependency-check plugin will complain to say there is a dependency convergence issue. This plugin isn't necessarily required, but is good because it forces us to make a decision on which version of the library we want/need. 
A good way to visualize this is to run `mvn package -DskipTests dependency:tree` which will show you all of our libraries and their nested dependencies.

Now to solve these dependency convergence issues, we can use the `<dependencyManagement>` section of the parent pom. This section doesn't actually declare dependencies, but rather says, if this dependency is included, either directory or transitively, use this version. With this dependency management section, we can guide maven along to find the proper mix of versions of our direct and transitive dependencies. 

Now to get why this changelog includes ant, that is because I wanted to bump our version of ant so that I could get a version that would work nicely with the newer Java EE dependencies I pulled in.

Reply:
With all of that said, this dependency convergence stuff is kinda a black art and there isn't necessarily a perfect solution. Sometimes I go through and remove all our "suggestions" in the `dependencyManagement` section and start over, but its a pain that I try to do infrequently.

Reply:
Thank you very much for the thorough explanation!

My comment was more about "Why do we have a dependency on `ant` given that we already have `maven`", but if it makes things easier, given that we have more than 200 dependencies, it doesn't make a big difference I guess :)

Reply:
Yep the dependency was already there, I merely bumped the version and moved the version enforcement to the parent pom.

Reply:
Apparently, ant isn't needed: #1041